vaapi/vaapidisplay.cpp

```diff
@@ -320,4 +333,14 @@ DisplayPtr VaapiDisplay::create(const NativeDisplay& display)
 {
     return DisplayCache::getInstance()->createDisplay(display);
 }
+
+DisplayPtr VaapiDisplay::create(const NativeDisplay& display, VAProfile profile)
+{
+    SharedPtr<DisplayCache> cache = DisplayCache::getInstance();
+    const std::string name="hybrid";
```

Comment:
Which driver should load is really a platform related info. Yami core library  do not need know this, and it not even possible. We'd better select driver name and init va in  in VaapiV4L2Device::Initialize() and set va display as NATIVE_DISPLAY_VA. This also apply to drm device name, open "/dev/card0" or ""/dev/dri/renderD128" should decide in VaapiV4L2Device::Initialize() not libyami core library.